nibabel/openers.py

```diff
@@ -9,19 +9,66 @@
 """ Context manager openers for various fileobject types
 """
 
-from os.path import splitext
-import gzip
 import bz2
```

Comment:
Okay. `bz2` sounds like its own issue, not something we can quickly take care of while we're here.